docs/development/git-workflow.md

```diff
@@ -0,0 +1,128 @@
+# Git Workflow
+
```

Comment:
I consider here is lack of something summing up the content of this article.🫤

Reply:
Good point, I will add an introduction later.

Reply:
Here we go! If you have any specific suggestions, just let me know.